package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Uglifies an HTML file and its associated CSS for compression. Great for HTML emails!",
   "main": "lib/main.js",
   "scripts": {
-    "test": "mocha test/main.js"
+    "test": "mocha test/main.js && node test/benchmark.js"
```

Comment:
Separate this into its own. 

This is not actionable on a CI if we wanted to do something else. Plus separation of concerns.